docs/user/datastores/query_interceptor.rst

```diff
@@ -136,3 +141,6 @@ An example is given here.  Durations can be given in a number of days, hours, or
         }
       }
     }
+
+To disable the guard on a per-environment basis, set the system property ``geomesa.guard.graduated.<typeName>.disable``
```

Comment:
I wonder how hard it'd be to sort out common configuration code to make it 'easy' / abstract for implementations to have this kind of capability....